framework/components/ecl-footers/_footers.scss

```diff
@@ -1,142 +1,122 @@
-/*
-Footer
-
-Site-wide footer component
-
-Markup: html/footer.html
-
-Style guide footer
-*/
-
-.footer {
-  background-color: $blue-medium;
+/**
+ * Footers component
+ * @define footers ; weak
+ */
+
+.ecl-footers__site-identity,
+.ecl-footers__site-corporate,
+.ecl-footers__ec {
+  @extend %hidden-print;
 }
 
-.footer-bottom {
-  background-color: $brand-primary;
-  padding: 0.9em 0;
+.ecl-footers__social-icon {
+  &::before {
+    margin-right: 8px !important;
+  }
+
+  &::after {
+    content: none !important;
+    display: none !important;
+  }
 }
 
-.footer,
-.footer-bottom {
-  @extend %hidden-print;
-  color: $white;
+.ecl-footers__title {
+  border-bottom: 2px solid map-get($colors, 'blue-50');
+  color: #fff;
+  margin-bottom: 1.1rem;
```

Comment:
Nope, I mean using `map-get($ecl-spacing, 'm')` or similar. Thus we use the same spacing everywhere (consistency matters)